src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoLazyAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto.ComponentModel.Attributes
+{
+    /// <summary>
+    /// The Ditto lazy property attribute.
+    /// Used for specifying that Ditto should lazy load this property.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
Id expect it to work in the same way as other attributes. I.E if I mark an enum with the enum processor then every time I declare that enum type as a property I know the processor will kick in without me having to declare the attribute again...

.... Damn your brain! I really like that you can declare all properties within a class as lazy. thinking about it that's a really neat way to do it.